.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: node_js CI
```

Comment:
Is this the most meaningful name? It can be anything I think

Reply:
This is copy/paste from a ci.yml file in frontend-enterprise. Will rename to something a bit more meaningful.